
--- build/gen-php/BeeswaxServiceClient.php
+++ build/gen-php/BeeswaxServiceClient.php
@@ -407,7 +407,7 @@
         throw new \Exception("get_results_metadata failed: unknown result");
     }

-    public function echo($s)
+    public function echo_str($s)
     {
         $this->send_echo($s);
         return $this->recv_echo();

--- build/gen-php/BeeswaxServiceIf.php
+++ build/gen-php/BeeswaxServiceIf.php
@@ -79,7 +79,7 @@
      * @param string $s
      * @return string
      */
-    public function echo($s);
+    public function echo_str($s);
     /**
      * Returns a string representation of the configuration object being used.
      * Handy for debugging.
